kong/plugins/ldap-auth/access.lua

```diff
@@ -84,14 +86,31 @@ local function load_credential(given_username, given_password, conf)
   return {username = given_username, password = given_password}
 end
 
+
+local function acl_cache_key(conf, username)
```

Comment:
Is this name intentional `acl_cache_key`, not a big deal though.

Reply:
oops... how did I miss that... good catch

Reply:
I copied that name from unit tests. I can rename both.